packages/components/src/components/dropdown-list/dropdown-list.tsx

```diff
@@ -146,14 +147,8 @@ const ListItems = React.forwardRef<HTMLDivElement, TListItems>((props, ref) => {
 });
 ListItems.displayName = 'ListItems';
 
-type TRef = {
-    dropdown_ref: React.RefObject<HTMLDivElement & SVGSVGElement> | null;
-    list_item_ref: React.RefObject<HTMLDivElement>;
-    list_wrapper_ref: React.RefObject<HTMLDivElement>;
-};
-
-const DropdownList = React.forwardRef<TRef, TDropDownList>((props, ref) => {
-    const { dropdown_ref, list_item_ref, list_wrapper_ref } = ref as unknown as TRef;
+const DropdownList = React.forwardRef((props: TDropDownList) => {
```

Comment:
```suggestion
const DropdownList = (props: TDropDownList) => {
```
We don't need `React.forwardRef` anymore since we don't use the `ref` from the second argument anymore. :)
I also have to ask you to check that the component is still working the same way as before when the app is launched with this change.🙏🏻